stable/sonatype-nexus/values.yaml

```diff
@@ -0,0 +1,55 @@
+# Default values for nexus.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: clearent/nexus
+  tag: 3.5.1-02
+service:
+  name: nexus
+  type: ClusterIP
```

Comment:
Wondering if this should be LoadBalancer by default to make it easier to use? Other web apps are currently doing this.

Reply:
Doesn't bother me one way or the other. Want me to switch it?

Reply:
I think it would provide a better UX yeah :)

Reply:
That would be nice, but this can be overwritten since it's part of `values.yaml` anyway.
Is it worth not merging the PR and wait just for this?

Reply:
Pushed the update